### PR TITLE
feat: add prixa tech

### PR DESCRIPTION
**Pull request by devkishor8007**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/prixa-tech.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+ "name": "Prixa Technologies",
+ "establishedAt": 1446014709,
+ "location": "Jawalakhel, Lalitpur, Nepal",
+ "website": "https://prixa.net",
+ "email": "info@prixa.org",
+ "phone": "9801970966",
```
Comment (Saphall) [inline data/prixa-tech.json:7]:
```suggestion
 "phone": "+977-9801970966",

```

File: data/prixa-tech.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ "socials": {
+   "facebook": "https://www.facebook.com/prixatech",
+   "github": null,
+   "instagram": null,
+   "linkedin": "https://www.linkedin.com/company/prixa",
+   "twitter": "https://twitter.com/prixatech",
+   "youtube": null
+ }
+}
```
Comment (Saphall) [inline data/prixa-tech.json:16]: It seems there is a new line at the end of the file in other json files. Let's follow the same format.

File: data/prixa-tech.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ "website": "https://prixa.net",
+ "email": "info@prixa.org",
+ "phone": "9801970966",
+ "socials": {
+   "facebook": "https://www.facebook.com/prixatech",
+   "github": null,
+   "instagram": null,
+   "linkedin": "https://www.linkedin.com/company/prixa",
+   "twitter": "https://twitter.com/prixatech",
+   "youtube": null
```
Comment (Saphall) [inline data/prixa-tech.json:14]:
```suggestion
   "youtube": "https://www.youtube.com/channel/UCEKpdNQryb_-kqfmolDknAw"

```
This seems like their youtube channel.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please make the changes requested by @Saphall !
Thank you :+1:

**Comment by devkishor8007** [comment]
@thenishantsapkota @Saphall please have a look, I did all changes as per the requests.

**Review by thenishantsapkota — APPROVED** [review]

**Review by thenishantsapkota — COMMENTED** [review]
LGTM!

**Review by Saphall — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contributions @devkishor8007 
 :partying_face:
Join our [Discord](http://discord.gg/7jwZaa8WDr) to claim your contributors role (_if you haven't already_) and stay connected with us.
Cheers!

**Pull request merged**